### PR TITLE
SuspendJob: graduate to beta

### PR DESCRIPTION
**Pull request by adtac**
KEP: https://github.com/kubernetes/enhancements/issues/2232

cc @soltysh @alculquicondor

### CODE REVIEW

**Comment by adtac** [comment]
Feature gate file was updated here: https://github.com/kubernetes/website/commit/372d34dc9467c9448a8e6a08a92e6b94260eeac5

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by adtac** [comment]
forgot to 1.21 -> 1.22 :sweat_smile:

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -395,15 +395,7 @@ Here, `W` is the number of work items.
...
-{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
-
-{{< note >}}
-Suspending Jobs is available in Kubernetes versions 1.21 and above. You must
-enable the `SuspendJob` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
-and the [controller manager](/docs/reference/command-line-tools-reference/kube-controller-manager/)
-in order to use this feature.
-{{< /note >}}
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/job.md:398]: People reading these docs might be running v1.21 or older. I recommend keeping a revised version of that text; maybe remind readers to check they're reading the docs for the version of Kubernetes they are running - you can link to https://kubernetes.io/docs/home/supported-doc-versions/ if that helps.

**Comment by adtac** [comment]
friendly ping @sftim @PI-Victor :)

**Review by sftim — COMMENTED** [review]
/lgtm

**Pull request merged**